scripts/smoke_tests/commands_test.ps1

```diff
@@ -29,12 +44,42 @@ $edge_deployment_metrics = "`"{'queries':{'mymetric':'SELECT deviceId from devic
 $edge_deployment_condition = "`"tags.environment='dev'`""
 $edge_module_content = "scripts/smoke_tests/edge_module_content.json"
 
-$dps_name = "smoketest-dps-$(New-Guid)"
 $dps_registration_id = "smoke-test-dps-registration"
 $dps_enrollment_id = "smoke-test-dps-enrollment"
 $dps_enrollment_group_id = "smoke-test-dps-enrollment-group"
 $dps_endorsement_key = "AToAAQALAAMAsgAgg3GXZ0SEs/gakMyNRqXXJP1S124GUgtk8qHaGzMUaaoABgCAAEMAEAgAAAAAAAEAibym9HQP9vxCGF5dVc1QQsAGe021aUGJzNol1/gycBx3jFsTpwmWbISRwnFvflWd0w2Mc44FAAZNaJOAAxwZvG8GvyLlHh6fGKdh+mSBL4iLH2bZ4Ry22cB3CJVjXmdGoz9Y/j3/NwLndBxQC+baNvzvyVQZ4/A2YL7vzIIj2ik4y+ve9ir7U0GbNdnxskqK1KFIITVVtkTIYyyFTIR0BySjPrRIDj7r7Mh5uF9HBppGKQCBoVSVV8dI91lNazmSdpGWyqCkO7iM4VvUMv2HT/ym53aYlUrau+Qq87Tu+uQipWYgRdF11KDfcpMHqqzBQQ1NpOJVhrsTrhyJzO7KNw=="
 
+$dt_instance_name = "smoketest-dt-$(New-Guid)"
+$dt_location = "westus2"
+$dt_eventgrid_endpoint = "smoketest-dt-eventgrid-endpoint"
+$dt_eventgrid_topic = "smoketest-dt-eventgrid-topic"
+$dt_eventgrid_secret = "https://accountname.blob.core.windows.net/containerName?sasToken"
```

Comment:
The service doesn't reject the invalid sas?

Reply:
Nope, it works as expected. The integration test **def test_dt_endpoints_routes** also uses the same secret.